package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "npm run build && npm run transactionTest && mocha test/packages/caver.utils.js && mocha test/packages/caver.klay.net.js && npm run accountTest && npm run serTest && npm run walletTest",
     "build-all": "gulp all",
-    "build": "./node_modules/.bin/webpack --mode production --entry ./webpack.js  --output dist/caver.min.js",
+    "build": "./node_modules/.bin/webpack --mode production",
```

Comment:
This is required to use caver-js on the browser. What makes you to remove this?

Reply:
I needed to add more configuration to fix build error. So i created webpack.config.js file separately.

Reply:
got it. Thanks.